public/static/docs/command-reference/checkout.md

```diff
@@ -133,15 +131,6 @@ The workspace looks almost like in this
     └── <code files here>
 ```
 
-We have these tags in the repository that represent different iterations of
-solving the problem:
-
-```dvc
-$ git tag
-baseline-experiment     <- first simple version of the model
```

Comment:
not sure what's going on here, see the comment below

Reply:
Extracted to #961

Reply:
so, let keep the description of tags? what was so wrong about this part?

Reply:
?

Reply:
Just noticed this. Checking... ⏳

Reply:
The output of `git tag` for this repo is much longer now because we have a tag for every commit, and these other ones, so I decided to just remove this. I'm restoring now with slightly different wording. See 76cb6914